sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/CreateNewEventPage.java

```diff
@@ -57,4 +58,36 @@
   public static final By CITY_INPUT = By.cssSelector(".popupContent #city");
```

Comment:
Please check the line 39, and fix my fault, I put `DISEASE_INPUT` instead `DISEASE_COMBOBOX`